management/Makefile

```diff
@@ -85,25 +96,37 @@ hydrate-kcc:
 	kustomize build -o ./.build/cnrm-install-services $(INSTANCE_DIR)/cnrm-install-services
 	kustomize build -o ./.build/cnrm-install-iam $(INSTANCE_DIR)/cnrm-install-iam
 
-# Install config connector in the cluster
+# The following steps install the config connector according to
+# https://cloud.google.com/config-connector/docs/how-to/advanced-install
 .PHONY: apply-kcc
 apply-kcc: hydrate-kcc
+	# Enable services required by config connector
 	anthoscli apply -f .build/cnrm-install-services
+	# Apply Google service account and workload identity binding
 	anthoscli apply -f .build/cnrm-install-iam
-	# Fisrt, apply and wait for the CRDs and namespaces
+	# Deploy Config Connector to the cluster
+	# Apply and wait for the CRDs and namespaces first
 	for resource in $$(find .build/cnrm-install-system -name '~g_v1_namespace_*.yaml' -o -name '*_customresourcedefinition_*.yaml'); \
 	do \
 		kubectl --context=$(MGMTCTXT) apply -f $$resource; \
 	done
-	# Then, apply all the resources
+	# Apply all the resources
 	kubectl --context=$(MGMTCTXT) apply -f .build/cnrm-install-system
 	# Wait until config connector operator pods are ready
 	kubectl wait -n configconnector-operator-system --for=condition=Ready pod --all --timeout 5m
-	# Wait for config connector pods are created
+	# Wait until config connector pods are created
+	# TODO(Bobgy): is there a better way than sleep?
```

Comment:
[ASK] Can you create an issue? I will pick it up.
I think we will have to monitor the status of the operator. That will do the job here.

Reply:
Thanks!
Updated https://github.com/kubeflow/gcp-blueprints/issues/159